pulsar-client/src/main/java/org/apache/pulsar/client/util/ExecutorProvider.java

```diff
@@ -18,40 +18,64 @@
  */
 package org.apache.pulsar.client.util;
 
-import static com.google.common.base.Preconditions.checkArgument;
-import static com.google.common.base.Preconditions.checkNotNull;
+import com.google.common.collect.Lists;
+import io.netty.util.concurrent.DefaultThreadFactory;
+import javafx.util.Pair;
```

Comment:
Do not use `javafx` import, rather `commons.lang3.`

Reply:
fixed